src/components/ResourceCard/ResourceCardBody/ImmunizationCardBody.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { shape } from 'prop-types';
+import CardBodyField from './CardBodyField';
+import {
+  getPatientAgeAtResourceDate,
+  getPrimarySource,
+  getStatus,
+} from '../../../resources/fhirReader';
+import CARD_BODY_LABEL from '../../../resources/cardBodyLabel';
+
+const ImmunizationCardBody = ({ resource, patientResource }) => {
+  const { type, subType } = resource;
+
+  return (
+    <>
+      <CardBodyField
+        label={CARD_BODY_LABEL.age}
+        value={getPatientAgeAtResourceDate(resource, patientResource)}
+      />
+      <CardBodyField
+        label={CARD_BODY_LABEL[type]}
+        value={subType}
+        bold
+      />
+      <CardBodyField
+        label={CARD_BODY_LABEL.given}
+        value={null}
+      />
+      <CardBodyField
+        label={CARD_BODY_LABEL.reported}
+        value={null}
+      />
+      <CardBodyField
+        label={CARD_BODY_LABEL.primarySource}
```

Comment:
I wouldn't show this information in a patient-facing app. Adds more confusion than help.